coalahtml/tree.py

```diff
@@ -0,0 +1,65 @@
+from collections import OrderedDict
+from queue import Queue
+import os
+
+
+class Node:
+
+    def __init__(self, path):
+        self.child = {}
+        self.parent = None
+        self.path = path
```

Comment:
IIUC this path is always either filename or dirname, so maybe you can use a better name.